packages/amplify-graphql-schema-generator/src/schema-representation/schema.ts

```diff
@@ -15,6 +15,7 @@ export class Schema {
   }
 
   public addModel(model: Model): void {
+    console.log(this.engine);
```

Comment:
Is this leftover from dev/debugging? Any reason a customer needs to see this?

Reply:
Yes its a leftover might have missed it I will remove it